letsencrypt/views/base_config_settings.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+
+    <record id="base_config_settings" model="ir.ui.view">
+        <field name="name">Letsencrypt base config settings</field>
+        <field name="model">base.config.settings</field>
+        <field name="inherit_id" ref="base_setup.view_general_configuration" />
+        <field name="arch" type="xml">
+            <xpath expr="./group[last()]" position="after">
+                <field name="letsencrypt_needs_dns_provider" invisible="1" />
+                <group
+                    name="letsencrypt"
+                    string="LetsEncrypt"
+                    attrs="{'required':
+                        [('letsencrypt_needs_dns_provider', '=', True)]}">
+                    <field name="letsencrypt_dns_provider" />
```

Comment:
organize this in a way that every provider has its own group, so that overrides can easily add their configuration values